docs/src/operate/c8y/connect.md

```diff
@@ -58,183 +59,236 @@ in this case we are told to run `tedge config set c8y.url <value>`.
 
 ## Making the cloud trust the device
 
-The next step is to have the device certificate trusted by Cumulocity. This is done by uploading the certificate of the signee.
-You can upload the root certificate using the [Cumulocity UI](https://cumulocity.com/docs/device-management-application/managing-device-data/#managing-trusted-certificates)
-or with [`tedge cert upload`](../../references/cli/tedge-cert.md) command as described below.
+%%te%% primarily uses a x509 certificate to connect to Cumulocity, therefore the certificate must be trusted by Cumulocity in order for the connection to be established.
 
-:::note
-The `tedge cert upload` command requires the credentials of a Cumulocity user
-having the permissions to upload trusted certificates on the Cumulocity tenant of the device.
-
-The user name is provided as `--user <username>` parameter,
-and the command will prompt you for this user's password.
-These credentials are used only for this upload and will in no case be stored on the device.
-:::
-
-<UserContext>
-
-```sh
-sudo tedge cert upload c8y --user "$C8Y_USER"
-```
-
-</UserContext>
+There are a few different ways to get the device certificate to be trusted by Cumulocity, below lists a few common options.
```

Comment:
We might say a word about a third option that is to get device certificate from a public CA and to upload the CA signing certificate to C8Y.

Reply:
Yes I would considering this, but couldn't get a reduced version which didn't go into the details of external CA, and how should the user then renew the certificate with them, as all of it really depends on the external CA.

Reply:
Okay, so let say nothing in that section.

Reply:
I agree that it would be worthwhile adding it, but it will take a bit longer to get a reasonable wording without getting too technical, so we can postpone that.